### PR TITLE
Add overlay for kcp

### PR DESCRIPTION
**Pull request by jinqi7**
Signed-off-by: Jing Qi <jinqi@redhat.com>

### CODE REVIEW

**Comment by Dannyb48** [comment]
I'm still waiting to get access to the staging cluster but @jinqi7 @srivickynesh @jsztuka  is anyone able to access and identify why this Pipeline is taking so long? It was started 9 hours ago and it is still in progress?

**Comment by Dannyb48** [comment]
So the pipeline is still running for some reason as a status here in GH but when I log in it shows the pipeline completed but failed all the sanity label tests we have.

**Comment by jinqi7** [comment]
> I'm still waiting to get access to the staging cluster but @jinqi7 @srivickynesh @jsztuka is anyone able to access and identify why this Pipeline is taking so long? It was started 9 hours ago and it is still in progress?

sanity-label-check-required_checks test FAIL:
"The required 'name' label is missing!"
"The required 'com.redhat.component' label is missing"
"The required 'version' label is missing!"
"The required 'description' label is missing!"
"The required 'io.k8s.description' label is missing!"
"The required 'vcs-ref' label is missing!"
"The required 'vcs-type' label is missing!"
"The required 'architecture' label is missing!"
"The required 'com.redhat.build-host' label is missing!"
"The required 'vendor' label is missing!"
"The required 'release' label is missing!"
"The required 'url' label is missing!"
"The required 'build-date' label is missing!"
"The required 'distribution-scope' label is missing!"
-------------
sanity-label-check-optional_checks test FAIL:
"The 'maintainer' label should be defined"
"The 'summary' label should be defined"

**Comment by srivickynesh** [comment]
@jinqi7, The image doesn't have all the information required to PASS, so you see the "Label missing" errors.
> {
    "Name": "*************************************************",
    "Digest": "******************************************************************",
    "RepoTags": [**************],
    "Created": 
    "DockerVersion": "",
    "Labels": { *******************************
    },
    "Architecture": "amd64",
    "Os": "linux",
    "Layers": [
    ],
    "Env": [
    ]
}
I'm Still digging into, why the pipeline as code still waiting for resources even after completing all the pipeline runs.

File: config/kcp-rbac/kustomization.yaml
Comment (jsztuka) [inline config/kcp-rbac/kustomization.yaml:15]: Where are those resource located? I can't find it within PR.

Comment (jinqi7) [reply config/kcp-rbac/kustomization.yaml]: They are removed and the kustomization.yaml was not updated to the newest one. Thanks!

Comment (jinqi7) [reply config/kcp-rbac/kustomization.yaml]: Updated

**Comment by jsztuka** [comment]
Issue in CI is not caused by changes in PR.

**Review by jsztuka — APPROVED** [review]
LGTM

**Comment by johnbieren** [comment]
It lgtm if these things are needed... I haven't found I need these things in testing the release-service on kcp

**Comment by dirgim** [comment]
looks good, but will first need to account for the RBAC additions in #28

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**
